### PR TITLE
Update FAQ, Command summary and assumptions about TA

### PR DESCRIPTION
**Pull request by Zack-Tay**
Update FAQ, Command summary and assumptions about Target audience.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -97,11 +97,10 @@ confidence to make full use of TAHelper's features.
...
       - It should look something like this:
-      
 
 5. Launching TAHelper
     - Type `java -jar tahelper.jar` command and hit Enter to run TAHelper.<br>
-    - It should look something like this (in this case my jar file is in a folder called tahelper): 
+    - It should look something like this (in this case my jar file is in a folder called `tahelper`):
     - ![cmd](images/cmdwinguide2.png)
     - A GUI similar to the below should appear in a few seconds.<br>
     - ![Ui](images/Ui.png) (to update!!).
```
Comment (mahadhir247) [inline docs/UserGuide.md:106]: I think we should remove the bullet points from the images since it was an issue brought up from PED

**Review by mahadhir247 — COMMENTED** [review]
LGTM just some formatting regarding bullet points and images

File: docs/UserGuide.md (hunk 1 above)
Comment (Jajared) [reply docs/UserGuide.md]: Yep @mahadhir247 no worries on this. I will update together with the UI in a seperate PR

**Review by Jajared — APPROVED** [review]
LGTM! Thanks for updating the command summary. I will have a comprehensive review on this in a seperate PR fyi

**Pull request merged**